frontend/src/app/patients/ManagePatients.jsx

```diff
@@ -19,8 +19,10 @@ import {
 const ajv = new Ajv({ allErrors: true });
 const validate = ajv.compile(schemaPatient.default);
 
+const patientQuery = gql`{patient{patientId,firstName,lastName,middleName,birthDate}}`;
+
 const ManagePatients = () => {
-  const patients = useSelector(getPatientsWithLastTestResult());
+  const {data, loading, error} = useQuery(patientQuery)
```

Comment:
worth a discussion with the expanded team, but we should figure out when the client is querying for information vs reading from the in memory store. 

I was roughly thinking the general practice would be:
1. app makes query on first render of the page
2. query response gets stored in redux
3. app re-renders based on new data in redux
4. to get updates, the app either is force refreshed (go to step 1), or subscribes to updates (go to step 2)

There are considerations around:
- minimize response sizes, via pagination or querying based on a lastUpdated value
- conflict resolutions (for offline writes)
- storage (if the client stores 100K test results in redux, well, that can be 100MB...)


Also, the `getPatientsWithLastTestResult()` has a tiny bit of business logic in it. Not sure if the "days since last test" column will work without it:
<img width="765" alt="Screen Shot 2020-11-12 at 12 23 19 PM" src="https://user-images.githubusercontent.com/35268641/98992339-ed281280-24e1-11eb-95ff-39f81fbcc28e.png">

Reply:
I would punt on the discussion until post MVP and just load all the data from the backend. will work for the 1-5 users and is significantly less complex

> Also, the getPatientsWithLastTestResult() has a tiny bit of business logic in it. Not sure if the "days since last test" column will work without it:

good point. working on adding dummy test results to the backend no and will pull this logic into the component